Objects/Converters/ConverterRhinoGh/ConverterRhinoGhShared/ConverterRhinoGh.Utils.cs

```diff
@@ -92,6 +92,39 @@ private string GetCommitInfo()
   private static string UserStrings = "userStrings";
   private static string UserDictionary = "userDictionary";
 
+  private void SetUserInfo(Base obj, ObjectAttributes attributes)
+  {
+    // set user strings
+    if (obj[UserStrings] is Base userStrings)
+    {
+      foreach (var member in userStrings.GetMembers(DynamicBaseMemberType.Dynamic))
+      {
+        attributes.SetUserString(member.Key, member.Value as string);
+      }
+    }
+
+    // set name or label
+    var name = obj["name"] as string ?? obj["label"] as string; // gridlines have a "label" prop instead of name?
+    if (name != null)
+    {
+      attributes.Name = name;
+    }
+
+    // set revit parameters as user strings
+    if (obj["parameters"] is Base parameters)
+    {
+      foreach (var member in parameters.GetMembers(DynamicBaseMemberType.Dynamic))
+      {
+        if (member.Value is Objects.BuiltElements.Revit.Parameter parameter)
+        {
+          var convertedParameter = ParameterToNative(parameter);
+          var paramName = $"{convertedParameter.Item1}({member.Key})";
+          attributes.SetUserString(paramName, convertedParameter.Item2);
+        }
+      }
+    }
+  }
```

Comment:
Given that these changes are replicated both in connector and converter, this could do with some `SharedResources` project in Rhino to hold all these types of shared implementations.

Since we're already looking at DI and DUI3, I don't think this is a blocker for this PR, but worth keeping in mind for the use-case